Project/Source/Components/UI/ComponentTransform2D.h

```diff
@@ -13,8 +13,45 @@
 #define TESSERACT_ENGINE_API __declspec(dllexport)
 #endif
 
-//TODO THIS CLASS CURRENTLY ONLY WORKS FOR OBJECTS THAT USE THE CENTER OF THE SCREEN AS THE CENTER OF COORDINATES,
-//NEITHER PIVOT NOR ANCHOR ARE TAKEN INTO ACCOUNT IN ANY WAY, NOR ARE THEY MODIFIABLE LIKE THEY WOULD IN UNITY
+// ANCHOR PRESETS CREATED BUT NOT WORKING -> WOLUD BE FIXED IN OTHER BRANCH
+// ANCHOR ARE TAKEN INTO ACCOUNT IN ANY WAY, NOR ARE THEY MODIFIABLE LIKE THEY WOULD IN UNITY
```

Comment:
Add a TODO here so that we can find this easily

Reply:
Ok! I will put it in the next commit ;)